### DISCUSSION TITLE
new submit button on form, is possible?

### DISCUSSION THREAD

**Discussion by fabioonet** [filamentphp/filament#6465] [General]
Hello,

Is possible duplicate submit button and position in the middle of form for example?

**Comment by fabioonet** [comment]
anyone can help me ?
thanks

**Comment by danharrin** [comment]
There is no built-in way to do it. You would need a `View::make()` with a view that renders an `<x-forms::button>` with `type="submit"`

**Reply by fabioonet** [reply]
Thank very much!
